cloudflare/resource_cloudflare_worker_script.go

```diff
@@ -184,24 +163,8 @@ func resourceCloudflareWorkerScriptDelete(d *schema.ResourceData, meta interface
 }
 
 func resourceCloudflareWorkerScriptImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	// split the id so we can lookup
-	idAttr := strings.SplitN(d.Id(), ":", 2)
-	var scriptType string
-	var scriptId string
-	if len(idAttr) == 2 {
-		scriptType = idAttr[0]
-		scriptId = idAttr[1]
-	} else {
-		return nil, fmt.Errorf("invalid id (\"%s\") specified, should be in format \"scriptType:scriptId\"", d.Id())
-	}
-
-	if scriptType == "name" {
-		d.Set("name", scriptId)
-	} else if scriptType == "zone" {
-		d.Set("zone_id", scriptId)
-	} else {
-		return nil, fmt.Errorf("invalid scriptType (\"%s\") specified, should be either \"name\" or \"zone\"", scriptType)
-	}
+	scriptID := d.Id()
```

Comment:
We may have a mismatch about ID format here. If the IDs follow the `name:{scriptName}` format, then would we still need people to specify `name:{scriptName}` when they import it?

Reply:
Dropped prefix.